### PR TITLE
mean and lean  Gopkg.toml

### PR DESCRIPTION
**Pull request by stappersg**
Removed boiler template lines

Added a last line marker

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -1,26 +1,12 @@
...
-# [[override]]
-#  name = "github.com/x/y"
-#  version = "2.4.0"
-
 
 [[constraint]]
   branch = "master"
   name = "golang.org/x/net"
+
+# l l
```
Comment (cobratbq) [inline Gopkg.toml]: I don't know this "last line marker", how does this work?

Comment (stappersg) [reply Gopkg.toml]: It is just another comment line, which is at the end of file.
`# Last Line` is too much typing.
`# L L` would mean pressing shift to get CAPS

Seeing the last line marker indicates that the final part of file was transmitted.
Also you can stop pressing "cursor down" or "page down".

**Comment by stappersg** [comment]
Last Line dropped.
But I'm not happy with it.

**Comment by stappersg** [comment]
preparing a clean patch

**Pull request closed without merging**